src/partners/dexAg/dexAg.js

```diff
@@ -376,6 +376,11 @@ export default class DexAg {
       : 'ag';
 
     const tradeDetails = await this.createTransaction(swapDetails, dexToUse);
+    if (tradeDetails.metadata.marketImpact) {
```

Comment:
`tradeDetails.metadata.marketImpact` this always exists
we have to make sure market impact is less than 10